aiida_kkr/tools/common_workfunctions.py

```diff
@@ -331,7 +337,9 @@ def get_inputs_common(calculation, code, remote, structure, options, label, desc
 
     if code:
         inputs.code = code
-
+        _sched = load_computer(code.computer.label).scheduler_type
```

Comment:
Why is `load_computer` needed? Does `code.computer` not give you the computer already?

Reply:
Yeah, you are right, I copied this from some other function where I wanted to load the computer for some tests. I have changed it as you recommed.